extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -179,13 +189,24 @@ export class AmpLightboxViewer extends AMP.BaseElement {
             const clonedNode = element.cloneNode(deepClone);
             clonedNode.removeAttribute('on');
             const descText = this.manager_.getDescription(element);
-            if (descText) {
-              clonedNode.descriptionText = descText;
+            const metadata = {
+              'descriptionText': descText,
```

Comment:
nit: typically we don't quote object literal keys. or was closure complaining about them?